edalize/yosys.py

```diff
@@ -51,27 +76,36 @@ def configure_main(self):
         for key, value in self.vlogparam.items():
             if type(value) is str:
                 value = "{\"" + value + "\"}"
-            _s = r"chparam -set {} {} {}"
+            _s = r"chparam -set {} {} \$abstract\{}"
```

Comment:
Rebase error. Edalize was recently updated to drop the old chparam syntax that used $abstract

Reply:
Fixed